libpince/debugcore.py

```diff
@@ -1315,8 +1315,9 @@ def get_stacktrace_info():
 
 
 #:tag:Stack
-def get_stack_info():
+def get_stack_info(from_base_pointer=False):
```

Comment:
`from_base_pointer` param isn't documented. You can check functions like `handle_signals` to see how args are documented. This parameter also needs a boolean type hint